chapters/connectors.tex

```diff
@@ -384,7 +384,7 @@ \section{Generation of Connection Equations}\label{generation-of-connection-equa
 \begin{lstlisting}[language=modelica]
 connect(a, b);
 \end{lstlisting}
-the primitive components of \lstinline!a! and \lstinline!b! form a connection set, together with an indication of whether they are from an inside or an outside connector.
+each of their primitive components will generate one connection set comprised of the corresponding primitive components of \lstinline!a! and \lstinline!b! together with an indication of whether they are from an inside or an outside connector.
```

Comment:
This is repetitive: primitive components will generate one ... set ...comprised of ... primitive components. Perhaps:
```suggestion
one connection set is generated for every pair of corresponding primitive components of \lstinline!a! and \lstinline!b! together with an indication of whether they are from an inside or an outside connector.
```

Reply:
Ah, yes, that's better.